.github/workflows/cli.yml

```diff
@@ -0,0 +1,59 @@
+name: cli
+
+on:
+  push:
+    branches:
+      - main
+  pull_request: {}
+
+concurrency:
+  group: cli-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build with Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '12', '14', '16' ]
```

Comment:
@lascenify by doing this we ensure that the CLI works with several versions of NodeJS. Otherwise, we might end up using an API that's only available in a recent version of Node and that breaks the CLI for users.